store/copr/coprocessor.go

```diff
@@ -569,7 +577,9 @@ func isSmallTask(task *copTask) bool {
 	// strictly, only RowCountHint == -1 stands for unknown task rows,
 	// but when RowCountHint == 0, it may be caused by initialized value,
 	// to avoid the future bugs, let the tasks with RowCountHint == 0 be non-small tasks.
-	return task.RowCountHint > 0 && task.RowCountHint <= CopSmallTaskRow
+	return task.RowCountHint > 0 &&
```

Comment:
A concern is that if the batch is enabled by default and the batched task could not use the coprocessor cache which means **the coprocessor is disabled by default for the batched tasks**, would it be a risk situation that this may cause regression?
In my understanding, the batched tasks are small ones and they should be processed fast usually it's not a problem

Reply:
Maybe we could support coprocessor cache for batched tasks in the future.

Reply:
> the coprocessor is disabled by default for the batched tasks

Yes, there is a low risk of regression by the disabled cache, the solution might be batch supporting cop cache. But at least we have the workaround, turn off batch so that the regression can be eliminated.

Reply:
Could it result in worse memory controllability because of disabling paging?

Reply:
I changed the batch builder to only handle small tasks, to avoid making the memory usage worse.

Reply:
> I changed the batch builder to only handle small tasks

This should prevent the only memory risk what I can imagine that the table is int handle and table lookup tasks are large ranges, after batch, each response comsumes more memory.